docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,146 @@
+
+### Create a new app {#create-app}
+
+Before you start developing with Bolt, you'll want to create a Slack app.
+
+To create the app, you'll run the following command:
+
+```bash
+slack create my-app -t slack-samples/bolt-python-starter-template -b future
+```
+
+This command creates an app through the CLI by cloning a specified template. In this case, the template is the <a href="https://github.com/slack-samples/bolt-python-starter-template/tree/future" target="_blank">Bolt for Python Starter Template</a> on the `future` branch. This starter template includes a "Hello World" example that demonstrates how to use <a href="/bolt-python/future/concepts#built-in-functions" target="_blank">built-in</a> and <a href="/bolt-python/future/concepts#functions" target="_blank">custom</a> Functions, <a href="https://api.slack.com/future/triggers" target="_blank">Triggers</a> and <a href="/bolt-python/future/concepts#manifest-workflows" target="_blank">Workflows</a>.
+
+Once the app is successfully created, you should see a message like this:
+
+```text
+✨ my-app successfully created
+
+🧭 Explore your project's README.md for documentation and code samples, and at any time run slack help to display a list of available commands
+
+🧑‍🚀 Follow the steps below to try out your new project
+
+1️⃣  Change into your project directory with: cd my-app
+
+2️⃣  Develop locally and see changes in real-time with: slack run
+
+3️⃣  When you're ready to deploy for production use: slack deploy
+
+🔔 If you leave the workspace, you won’t be able to manage any apps you’ve deployed to it. Apps you deploy will belong to the workspace even if you leave the workspace
+```
+
+---
+
+### Set up your trigger {#setup-trigger}
+
+As mentioned, this app comes with pre-existing functionality - it uses Functions, Workflows and a Trigger set up to kick things off. Let's run a command to initialize that Trigger via the CLI.
+
+First, make sure you're in the project directory in your command line: `cd my-app`
+
+Then, run the following command to create a Trigger:
+
+```bash
+slack triggers create --trigger-def "triggers/sample-trigger.json"      
+```
+
+The above command will create a trigger for the selected workspace. Make sure to select the workspace you want. Once the trigger is successfully created, you should see an output like this:
+
+```bash
+⚡ Trigger created
+   Trigger ID:   [ID]
+   Trigger Type: shortcut
+   Trigger Name: Sample Trigger
+   URL: https://slack.com/shortcuts/[ID]/[Some ID]
+```
+
+The provided URL will be what you use to run your Trigger. Copy this URL and save it somewhere; you'll need it for later.
+
+---
+
+### Run your app {#run-your-app}
+
+Now that your app and Trigger are successfully created, let's try running it!
+
+```bash
+# install the required project dependencies
+pip install -r requirements.txt
+
+# start a local development server
+slack run
+```
+
+Executing `pip install -r requirements.txt` installs all the project requirements to your machine.
```

Comment:
We may want to link to any existing documentation we have around setting up a virtual env, as that is the recommended way of working in Python projects.

Reply:
Yeah I've been thinking about doing this in our documentation, currently this is how we do it in the rest of our docs

Reply:
Let's chalk it up as a separate issue so we don't forget, and if/when we get to it we can update all the documentation (new platform or otherwise) to link to it where it makes sense.

Reply:
I created issue #777 to keep track of this